<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TeslaWebsiteChecker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="checkBox_all">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select All</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_na_US">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>United States</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_na_CA">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Canada</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_na_MX">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mexico</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_na_PR">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Puerto Rico</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>North America</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Europe</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_BE">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Belgium</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_CZ">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Czech</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_DK">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Denmark</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_DE">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>420</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Germany</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_GR">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>450</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Greece</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_ES">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>450</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Spain</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_HR">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Croatia</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_HU">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>480</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hungary</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_IE">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>540</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ireland</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_LU">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Luxembourg</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_NL">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Netherlands</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_IS">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>510</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Iceland</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_IT">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Italy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_NO">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>300</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Norway</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_AT">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Austria</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_PL">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>330</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Poland</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_PT">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Portugal</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_RO">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>390</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Romania</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_SL">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>420</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Slovenija</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_CH">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Switzerland</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_SE">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>480</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sweden</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_FI">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Finnland</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_GB">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>540</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>United Kingdom</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_na">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select All North America</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select All Europe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Select one or more countries to monitor</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_eu_FR">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>France</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>640</y>
      <width>661</width>
      <height>111</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>659</width>
       <height>109</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_output">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
